packages/tools/utils/src/any-boolean.js

```diff
@@ -188,6 +188,39 @@ true
  */
 export const isNotNull = _.not(_.isNull);
 
+/**
+ * Return true is the input is a function
```

Comment:
Typo:
```
* Return true if the input is a function
              ^
```

Reply:
thanks it was spread all over the file, fixed!